test/components/contextMenuTest.js

```diff
@@ -0,0 +1,106 @@
+/* global describe, it, before */
+
+const Brave = require('../lib/brave')
+const {urlInput} = require('../lib/selectors')
+
+describe('ContextMenu', function () {
+  function * setup (client) {
+    yield client
+      .waitForUrl(Brave.newTabUrl)
+      .waitForBrowserWindow()
+      .waitForEnabled(urlInput)
+  }
+
+  describe('navigation', function () {
+    Brave.beforeAll(this)
+
+    before(function *() {
+      yield setup(this.app.client)
+      this.formfill = Brave.server.url('formfill.html')
+      this.input = '[name="01___title"]'
+      this.values = ['Value 1', 'Value 2', 'Value 3', 'Value 4']
+
+      yield this.app.client
+        .tabByIndex(0)
+        .loadUrl(this.formfill)
+        .waitForVisible('<form>')
+
+      for (let i = 0; i < this.values.length; i++) {
+        yield this.app.client
+          .click(this.input)
+          .keys(this.values[i])
+          .click('#submit')
+          .waitForVisible('<form>')
+      }
+    })
+
+    it('check if context menu exists', function *() {
+      yield this.app.client
+        .tabByIndex(0)
+        .loadUrl(this.formfill)
+        .waitForVisible('<form>')
+        .click(this.input)
+        .click(this.input)
+        .windowByUrl(Brave.browserWindowUrl)
+        .waitForVisible('.contextMenuItemText')
+    })
+
+    it('click on item', function *() {
+      yield this.app.client
+        .tabByIndex(0)
+        .loadUrl(this.formfill)
+        .waitForVisible('<form>')
+        .click(this.input)
+        .click(this.input)
+        .windowByUrl(Brave.browserWindowUrl)
+        .waitForVisible('.contextMenu')
+        .click('.contextMenuItem')
+        .tabByUrl(this.formfill)
+        .getValue(this.input).should.eventually.be.equal(this.values[0])
+    })
+
+    it('select item via click and keys', function *() {
+      yield this.app.client
+        .tabByIndex(0)
+        .loadUrl(this.formfill)
+        .waitForVisible('<form>')
+        .click(this.input)
+        .click(this.input)
+        .windowByUrl(Brave.browserWindowUrl)
+        .waitForVisible('.contextMenu')
+        .keys('\uE015')
+        .pause(10)
+        .keys('\uE015')
+        .pause(10)
+        .keys('\uE007')
+        .pause(10)
+        .tabByUrl(this.formfill)
+        .getValue(this.input).should.eventually.be.equal(this.values[1])
+    })
+
+    it('select item via keys only', function *() {
+      yield this.app.client
+        .tabByIndex(0)
+        .loadUrl(this.formfill)
+        .waitForVisible('<form>')
+        .click(this.input)
+        .windowByUrl(Brave.browserWindowUrl)
+        .keys('\uE015')
+        .waitForVisible('.contextMenu')
+        .keys('\uE015')
+        .pause(10)
+        .keys('\uE015')
+        .pause(10)
+        .keys('\uE015')
+        .pause(10)
+        .keys('\uE015')
+        .pause(10)
+        .keys('\uE013')
+        .pause(10)
+        .keys('\uE007')
```

Comment:
Ok, the only thing that is now not working, and that's why test are failing, is enter. Enter is not prevented and that's why form is refreshed and test fails. Although it's strange, because event is prevented in contextMenu. I added test that check's if enter is prevented or not.